gubernator/templates/pr_dashboard.html

```diff
@@ -22,6 +22,7 @@
 		% continue
 	% endif
 	<h3 class="pr-section">
+	<a id="section-{{title|slugify}}" />
```

Comment:
Last thing: make this <a></a>-- Closing tags like this doesn't work in HTML5 https://stackoverflow.com/a/3558200